app/src/main/java/com/github/sdpcoachme/firebase/database/FireDatabase.kt

```diff
@@ -37,6 +38,17 @@ class FireDatabase(databaseReference: DatabaseReference) : Database {
         return getChild(accounts, userID).thenApply { it.exists() }
     }
 
+    override fun addEventsToDatabase(email: String, events: List<Event>): CompletableFuture<Void> {
+        return this.getUser(email).thenAccept {
+            val updatedUserInfo = if (it.events.isEmpty()) {
+                it.copy(events = events)
+            } else {
+                it.copy(events = it.events + events)
+            }
+            addUser(updatedUserInfo)
```

Comment:
The if else could probably be simplified into :
val updatedUserInfo = it.copy( it.events + events)